packages/widget-v2/src/components/RenderWalletList.tsx

```diff
@@ -83,12 +82,12 @@ export const RenderWalletList = ({
                   height={35}
                   width={35}
                   style={{ objectFit: "cover" }}
-                  src={imageUrl}
-                  alt={`${name} logo`}
+                  src={imageUrl ?? "https"}
```

Comment:
I don't get the "https" fallback, it'll show an empty image just like an empty string right? no big deal just seems random